### ISSUE TITLE
Deprecate `skipReleaseLabels` 

### ISSUE THREAD

**Issue by hipstersmoothie** [intuit/auto#541]
With the ability to specify an array of labels for any of the default labels `skipReleaseLabels`  is unnecessary

**Comment by zephraph** [comment]
Cool, so while not exactly what I'd proposed in https://github.com/intuit/auto/issues/476, this is definitely heading in a positive direction. 

Just so I understand...

There's now `major`, `minor`, `patch`, and `skip-release` sections in `labels` that can have multiple different labels. Does it make sense to support any other labels?

**Comment by hipstersmoothie** [comment]
I'd still like to be able to have custom changelog title labels. I think it still makes sense to support them, but maybe more in the way that you describe in #476

**Comment by zephraph** [comment]
@hipstersmoothie, the thing I'm struggling to wrap my head around is how does (or should) a label that's only responsibility is changing the changelog interact with labels whose job it is to do a release... And what's the value in having those two things as fundamentally different? 

So if you wanted to say... have a typescript changelog label. When you add that label, does it just create a duplicate section in the changelog with typescript grouped under it? Or does it override the other label. If it overrides, wouldn't it be weird to have a typescript section in the changelog that could've resulted in a major, minor, or patch release? That just feels... weird to me. 

Conceptually I just feel like it'd be simpler if we had only four types of releases, but you could have multiple of those releases (and the changelog titles could be altered for each). 

So if you have a typescript label, it could set the changelog title to typescript and do a minor release.

**Comment by hipstersmoothie** [comment]
> When you add that label, does it just create a duplicate section in the changelog with typescript grouped under it? Or does it override the other label. 

Overrides the other label but is organized in the same order prefereably

```
# Major
# Minor 
# Typescript (minor)
# Patch
```

> Conceptually I just feel like it'd be simpler if we had only four types of releases, but you could have multiple of those releases (and the changelog titles could be altered for each).

What would this type of changelog look like?

**Comment by zephraph** [comment]
The changelog would look the same as it does today. Just given your above example you'd have to have...

```
labels: [
  {
    label: "New Types",
    releaseType: "minor",
    changelogTitle: "Typescript (minor)"
  }
]
```

Idk, ha. This changelog stuff is pretty hairy. I'm good with whatever, but ideally we just keep it as simple as possible.

**Comment by adierkens** [comment]
:rocket: Issue was released in v8.0.0-next.8 :rocket:

**Comment by adierkens** [comment]
:rocket: Issue was released in v8.0.0 :rocket:
